homeassistant/components/airnow/__init__.py

```diff
@@ -44,10 +44,12 @@
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up AirNow from a config entry."""
-    api_key = entry.data[CONF_API_KEY]
-    latitude = entry.data[CONF_LATITUDE]
-    longitude = entry.data[CONF_LONGITUDE]
-    distance = entry.data[CONF_RADIUS]
+    api_key = entry.data.get(CONF_API_KEY)
+    latitude = entry.data.get(CONF_LATITUDE)
+    longitude = entry.data.get(CONF_LONGITUDE)
```

Comment:
Why are we changing this?

Reply:
Consistency with the `.get(..., entry.options.get())` I originally added but it's been reverted now